mijia-homie/src/main.rs

```diff
@@ -513,18 +513,18 @@ async fn handle_bluetooth_event(
                         homie.remove_node(&sensor.node_id()).await?;
                     } else {
                         println!(
-                            "{} ({:?}) disconnected but was connected as {:?}.",
+                            "{} ({}) disconnected but was connected as {}.",
                             sensor.name, id, connected_id
                         );
                     }
                 } else {
                     println!(
-                        "{} ({:?}) disconnected but wasn't known to be connected.",
+                        "{} ({}) disconnected but wasn't known to be connected.",
                         sensor.name, id
                     );
                 }
             } else {
-                println!("Unknown device {:?} disconnected.", id);
```

Comment:
let me guess: the debug representation of Path is mental?

Reply:
It's not terrible, but something like DeviceId { Path { "/org/bluez/hci0/dev_11_22_33_44_55_66" } } seems unhelpfully verbose for a log message, as a user of the service doesn't really care how we represent things internally. The adapter and MAC address are the parts which are likely to be useful, so it seems nicer to just print that.